rules/windows/process_creation/win_webshell_detection.yml

```diff
@@ -16,20 +16,20 @@ logsource:
     product: windows
 detection:
     selection:
-        ParentImage:
-            - '*\apache*'
-            - '*\tomcat*'
-            - '*\w3wp.exe'
-            - '*\php-cgi.exe'
-            - '*\nginx.exe'
-            - '*\httpd.exe'
-        CommandLine:
-            - '*whoami*'
-            - '*net user *'
-            - '*ping -n *'
-            - '*systeminfo'
-            - '*&cd&echo*'
-            - '*cd /d*'  # https://www.computerhope.com/cdhlp.htm
+        ParentImage|endswith:
+            - '\apache*'
+            - '\tomcat*'
+            - '\w3wp.exe'
+            - '\php-cgi.exe'
+            - '\nginx.exe'
+            - '\httpd.exe'
+        CommandLine|contains:
+            - 'whoami'
+            - 'net user '
+            - 'ping -n '
+            - 'systeminfo'
```

Comment:
better split the executables from commandline

Reply:
Fixed